lms/static/js/search/views/search_item_view.js

```diff
@@ -25,9 +30,36 @@ define([
         render: function () {
             this.$el.html(this.tpl(this.model.attributes));
             return this;
+        },
+
+        /**
+         * Redirect to a URL.  Mainly useful for mocking out in tests.
+         * @param  {string} url The URL to redirect to.
+         */
+        redirect: function(url) {
+            window.location.href = url;
+        },
+
+        logSearchItem: function(event) {
+            event.preventDefault();
+            var target = this.model.id;
+            var link = $(event.target).attr('href');
+            var collection = this.model.collection;
+            var page = collection.page;
+            var pageSize = collection.pageSize;
+            var searchTerm = collection.searchTerm;
+            var index = collection.indexOf(this.model);
+            Logger.log("edx.course.search.result_selected",
+                {
+                    "search_term": searchTerm,
+                    "result_position": (page * pageSize + index),
+                    "result_link": target
+                });
+            this.redirect(link);
```

Comment:
This makes me a little nervous since the default behavior of `Logger.log` is to make an async HTTP request. I'm not a JS expert, so I'm not sure if we are gauranteed to always complete that request before the redirect, but I suspect that we are not.

IIRC `Logger.log` returns a jQuery deferred object, so you probably need to do something like:

```javascript
Logger.log("edx.course.search.result_selected",
{
    "search_term": searchTerm,
    "result_position": (page * pageSize + index),
    "result_link": target
}).resolve();
```

Or something similar to ensure that the AJAX request completes.